config/app-types.yml

```diff
@@ -16,7 +16,7 @@ appTypes:
   - title: Single-Page App
     name: spa
     description: "JavaScript web app that runs in the browser"
-    example: "e.g., AngularJS + Node.js, React"
+    example: "e.g., Angular + Node.js, React"
```

Comment:
Now that we are changing this line, I am wondering what the purpose is of `Node.js` here? Even more so why it is mentioned with Angular but both with React?

These are probably not questions for you @santoshyadavdev , not sure who can answer them.

Eitherway, As this is about SPA's, I think it makes more sense to use `e.g., Angular, React`.

Reply:
I added a commit to remove NodeJS here as well.